src/ecephys_etl/modules/vbn_create_stimulus_table/create_stim_table.py

```diff
@@ -0,0 +1,551 @@
+from functools import partial
+from typing import List, Tuple, Union
+import logging
+
+import numpy as np
+import pandas as pd
+
+from ecephys_etl.data_extractors.sync_dataset import Dataset
```

Comment:
`Dataset` is the baseclass for `EcephysDataset`. Given that this is in the tree under ecephys, wouldn't we typically be working with the latter?

Reply:
If we end up working with eye tracking in this repository, then the base `Dataset` class will still be needed as the `EcephysDataset` will (likely?) be too specific.